### PR TITLE
Next nyc meetup

### PR DESCRIPTION
**Pull request by xuv**
Updating previous events and adding the next NYC meetup.

### CODE REVIEW

File: _posts/2018-05-24-open-source-design-nyc.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+categories: design meetup community
+eventDate: Thu, 24 May 2018
+location: New York
+time: 6:00pm - 8.00pm (EDT)
+status: upcoming
+---
+
+Please join us on Thursday evening for a presentation and Q&A on [Cita](http://citapress.org) with Juliana Castro.
+
+[Juliana Castro](http://julianacastro.co/) is a graphic designer, art director, and open access enthusiast specializing in editorial and interaction design. Juliana has experience working in museums, educational institutions, and advertising agencies. She's the founder of cita, a feminist digital library and press publishing open-licensed books written by women. Juliana received a Fulbright Scholarship in 2016 and earned her M.F.A. at the University of Texas at Austin in 2018. During the summer of 2018, she’ll join the Berkman Klein Center for the Internet at Harvard University as the Harvard Open Access Project Intern. Juliana cares about the Internet, diversity, accessibility, and GIFs.
```
Comment (evalica) [inline _posts/2018-05-24-open-source-design-nyc.md]: "founder of Cita" I think it needs capitalization.

File: _posts/2018-05-24-open-source-design-nyc.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The meetup will take place in [ThoughtWorks offices, 99 Madison Avenue, 15th fl](https://www.google.com/maps/place/ThoughtWorks,+Inc./@40.7446828,-73.9870632,17z/data=!4m5!3m4!1s0x89c259a7c4fab243:0x7a7b1b141a048edf!8m2!3d40.7446828!4d-73.9848745) on **Wednesday 24th of May, from 6.00pm to 8.00pm**.
```
Comment (evalica) [inline _posts/2018-05-24-open-source-design-nyc.md]: 24th May is on a Thursday

**Review by evalica — APPROVED** [review]
Small fixes needed

**Comment by xuv** [comment]
@evalica Thx so much for looking at this. It's corrected.

**Comment by davelab6** [comment]
Great!

**Pull request merged**